target/linux/realtek/rtl838x/target.mk

```diff
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: GPL-2.0-only
 ARCH:=mips
 SUBTARGET:=rtl838x
-CPU_TYPE:=4kec
+CPU_TYPE:=24kc
```

Comment:
I actually cleaned up this stuff and moved to a unified config here https://gitlab.com/olliver/openwrt/openwrt/-/commit/7ab9cb4cdea1e2790119bd7cd5151fefc9654396

however, the rtl838x remains sticking out, as it isn't 34kc as the others are. Do we know for certain the rtl838x is a 24kc and not a 4kec? I know rtl839x and up is a 34kc.

So in itself, I think this is a good change, if it is true, which I don't know ...

Reply:
RTL8380M, RTL8381M and RTL8382M are 4KEc.

https://www.realtek.com/en/products/communications-network-ics/item/rtl8380m-vb-cg
https://www.realtek.com/en/products/communications-network-ics/item/rtl8381m-vb-cg
https://www.realtek.com/en/products/communications-network-ics/item/rtl8381m-vb-cg-2